tests/typ/layout/par-justify-cjk.typ

```diff
@@ -0,0 +1,21 @@
+// Test Chinese text in narrow lines.
```

Comment:
could you wrap everything at column 80 and indent the text in the text in the function calls with 2 spaces?

Reply:
Fixed in 1f55515329d790c405b3ed68dcd324023e0daffe